lib/src/commands/create.dart

```diff
@@ -14,7 +14,12 @@ const _defaultOrgName = 'com.example.verygoodcore';
 const _defaultDescription = 'A Very Good Project created by Very Good CLI.';
 final _defaultTemplate = CoreTemplate();
 
-final _templates = [_defaultTemplate, DartPkgTemplate(), FlutterPkgTemplate()];
+final _templates = [
+  _defaultTemplate,
+  DartPkgTemplate(),
+  FlutterPkgTemplate(),
+  FlutterPluginTemplate(),
```

Comment:
org_name that is being provided to this template is different than the org_name format the template is expecting. We should probably update the template to use the existing org_name format.